CMakeLists.txt

```
@@ -203,7 +218,12 @@ if(CHIRA_BUILD_EDITOR)
     if(WIN32)
         list(APPEND CHIRA_EDITOR_SOURCES ${CMAKE_CURRENT_LIST_DIR}/resources/editor/icon/icon.rc)
     endif()
-    add_executable(ChiraEditor ${CHIRA_EDITOR_SOURCES})
+    # macos bundle for editor
+    if(${CMAKE_SYSTEM} MATCHES "Darwin")
+        add_executable(ChiraEditor MACOSX_BUNDLE ${CHIRA_EDITOR_SOURCES})
+    else()
+        add_executable(ChiraEditor ${CHIRA_EDITOR_SOURCES})
+    endif()
```

This actually doesn't have any ill effect on alternate platforms
```suggestion
    add_executable(ChiraEditor MACOSX_BUNDLE ${CHIRA_EDITOR_SOURCES})
```

---

this could cause unintended side effects that we don't know of at this moment in time and as such will NOT be changing this

---

On any system besides Mac it does nothing according to cmake docs